public/js/components/AtomEdit/CustomEditors/AudioFields/AutomaticDataFetch.js

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from 'react';
+import {ManualDataInput} from "./ManualDataInput";
+
+class AutomaticDataFetch extends React.Component {
+
+  static propTypes = {
+    getAudioPageData: PropTypes.func,
+    onUpdate: PropTypes.func.isRequired,
+    onFormErrorsUpdate: PropTypes.func,
+    error: PropTypes.string,
+    audioPageData: PropTypes.object,
+    atom: atomPropType.isRequired,
+  };
+
+  state = {
+    audioPageUrl: '',
+    urlError: ''
+  };
+
+  getAudioDataFromCapi () {
+    if(this.checkUrlIsAudioPage(this.state.audioPageUrl)){
+      this.props.getAudioPageData(this.state.audioPageUrl, this.props.atom);
+      this.setState({urlError: ''})
+    } else {
+      this.setState({urlError: 'The url is not a valid Guardian audio page'})
+    }
+  }
+
+  checkUrlIsAudioPage (url) {
+    return url.startsWith('https://www.theguardian.com/news/audio');
```

Comment:
I don't think this is exhaustive. I can at least see `australia-news/audio` too, perhaps there are others. I don't know how to find them all though 😭

Reply:
ah... good catch. 
I don't want to get too fancy here. But can do:
url.startsWith('https://www.theguardian.com') && url.includes('audio')

Not perfect, but just want to catch 80% of errors

Reply:
Yeah let's try that!